examples/cms-contentstack/README.md

```diff
@@ -0,0 +1,131 @@
+# Create a Static Blog Site Using Next.js and Contentstack
+
+This example demonstrates how you can set up and run a server side example using Next.js and Contentstack with minimal steps. [SSR](https://nextjs.org/docs/basic-features/pages) and [Contentstack-CMS](https://www.contentstack.com/) as the data source.
+
+## Demo
+
+[https://cms-contentstack-starter.vercel.app/](https://cms-contentstack-starter.vercel.app/)
+
+## Deploy on Your Own
+
+Once you have access to [the required environment variables](#step-4-set-up-environment-variables), deploy the sample app using [Vercel](https://vercel.com?utm_source=github&utm_medium=readme&utm_campaign=next-example):
+
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fvercel%2Fnext.js%2Ftree%2Fcanary%2Fexamples%2Fcms-contentstack&env=CONTENTSTACK_API_KEY,CONTENTSTACK_DELIVERY_TOKEN,CONTENTSTACK_ENVIRONMENT,CONTENTSTACK_MANAGEMENT_TOKEN,CONTENTSTACK_API_HOST,CONTENTSTACK_APP_HOST,CONTENTSTACK_LIVE_PREVIEW&envDescription=For%20more%20about%20environment%20tokens&envLink=https%3A%2F%2Fwww.contentstack.com%2Fdocs%2Fdevelopers%2Fsample-apps%2Fbuild-a-starter-website-using-next-js-and-contentstack%23build-and-configure-the-website&project-name=cms-contentstack-starter&repo-name=contenstack-nextjs-starter&demo-title=cms-contentstack-starter&demo-description=Contentstack%20starter%20app%20example%20with%20Live%20Preview&demo-url=https%3A%2F%2Fcms-contentstack-starter.vercel.app%2F&demo-image=https%3A%2F%2Fimages.contentstack.io%2Fv3%2Fassets%2Fblt8f177c3a5bf3b9a0%2Fblt8d4529a8c7873223%2F6216200c6dad8d31f18e4603%2Fnextjs-starter-app.png)
+
+## Steps for Execution
+
+Execute [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app) with [npm](https://docs.npmjs.com/cli/init) or [Yarn](https://yarnpkg.com/lang/en/docs/cli/create/) to bootstrap the app:
+
+```bash
+
+npx create-next-app --example cms-contentstack cms-contentstack-app
+
+# or
+
+yarn create next-app --example cms-contentstack cms-contentstack-app
+```
+
+## Configuration
+
+### Prerequisites
+
+- Contentstack Account
+- Node.js version 12 or later (recommended 14)
```

Comment:
Since v12 is EOL, this light is probably unnecessary. See: https://endoflife.date/nodejs
```suggestion
```

Reply:
removed this line